oe_theme.theme

```diff
@@ -644,6 +641,26 @@ function oe_theme_preprocess_paragraph__oe_content_row__variant_inpage(array &$v
   $variables['links'] = $links;
 }
 
+/**
+ * Implements hook_preprocess_pattern().
+ */
+function oe_theme_preprocess_pattern_date_block(array &$variables): void {
+  // This make sure that the variant is not empty and has a "default" value.
+  // @todo: How about doing this in all components ?
+  // @todo: We should optimize how to compute the list of variants.
```

Comment:
Comment should rather be:
```
// This make sure that the variant value will always fallback to "default" if an empty or not valid value is passed.
```

I'd also remove the todo here since it's not something to do about the current block of code, I'd rather open a follow up to do this on all components.

Reply:
Updating now.